### ISSUE TITLE
ITKv4 build

### ISSUE THREAD

**Issue by T4mmi** [T4mmi/ITKBinaryThinning3D#7]
Hi there,

With the new version X.0.2, I lift some of the type limitations (wrapping macro), and evertything runs smooth, BUT:

**The ITKv4 branch won't properly build**, it passed build&tests but not the [packaging](https://circleci.com/gh/T4mmi/ITKBinaryThinning3D/213) step...

@thewtex do you have any hints about it ?

```
[2/16] Performing download step (download, verify and extract) for 'castxml'
FAILED: Wrapping/Generators/CastXML/castxml-prefix/src/castxml-stamp/castxml-download 
cd /work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src && /usr/bin/cmake -P /work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src/castxml-stamp/download-castxml.cmake && /usr/bin/cmake -P /work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src/castxml-stamp/verify-castxml.cmake && /usr/bin/cmake -P /work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src/castxml-stamp/extract-castxml.cmake && /usr/bin/cmake -E touch /work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src/castxml-stamp/castxml-download
-- Downloading...
   dst='/work/_skbuild/linux-x86_64-2.7/cmake-build/Wrapping/Generators/CastXML/castxml-prefix/src/method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
   timeout='none'
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
-- Retrying...
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
-- Retry after 5 seconds (attempt #2) ...
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
-- Retry after 5 seconds (attempt #3) ...
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
-- Retry after 15 seconds (attempt #4) ...
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
-- Retry after 60 seconds (attempt #5) ...
-- Using src='https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz'
CMake Error at castxml-stamp/download-castxml.cmake:159 (message):
  Each download failed!

    error: downloading 'https://midas3.kitware.com/midas/api/rest?method=midas.bitstream.download&checksum=84c2851a4df419b5834114bbfafa8d3e&name=castxml-linux.tar.gz' failed
         status_code: 22
         status_string: "HTTP response code said error"
         log:
         --- LOG BEGIN ---
           Trying 50.58.123.181...
```

**Comment by T4mmi** [comment]
PS : Got this same error  with both CircleCI and TravisCI, Appveyor does run but no artifacts are produced

**Comment by thewtex** [comment]
This [has been fixed](https://github.com/InsightSoftwareConsortium/ITK/commit/716fe079a6fad64f63f1166c7cb1565acef982ee#diff-93909ee5c75b85d1412947c92cef96cc) is the `release-4.13` branch, and it will be available for Python builds when [4.13.2 is released in a few weeks](https://github.com/InsightSoftwareConsortium/ITK/milestone/3).
